rules-tests/CodeQuality/Rector/Switch_/SingularSwitchToIfRector/Fixture/default_only.php.inc

```diff
@@ -10,6 +10,7 @@ final class DefaultOnly
         switch ($value) {
             default:
                 $result = 1000;
+                break;
```

Comment:
please don't change existing fixture, the code already valid. instead, create new one for it.

Reply:
done